### PR TITLE
Update instructions to include details on publishing NPM package.

### PR DESCRIPTION
**Pull request by JoshuaGross**

### CODE REVIEW

**Comment by JoshuaGross** [comment]
@mfine  @mookerji

File: Makefile [hunk 1]
```diff
@@ -82,6 +82,7 @@ javascript:
                        -r $(SBP_MAJOR_VERSION).$(SBP_MINOR_VERSION) \
 		       --javascript;\
 	cd $(SWIFTNAV_ROOT);
+	- npm version "v$(SBP_MAJOR_VERSION).$(SBP_MINOR_VERSION).$(SBP_PATCH_VERSION)" --no-git-tag-version
```
Comment (mfine) [inline Makefile]: what does the leading dash do?

**Comment by mfine** [comment]
Looks good :+1:

File: Makefile (hunk 1 above)
Comment (JoshuaGross) [inline Makefile]: whoa, eagle eyes! It ignores any errors from that line. In this case, `npm version` will throw an error if the version doesn't change, which we don't really care about.

Comment (mfine) [inline Makefile]: cool!

**Pull request merged**